scripts/release_info.py

```diff
@@ -0,0 +1,222 @@
+
+
+def _gather_logs(start, stop="HEAD"):
+    """Gathers the logs between the start and endpoint.
+
+    Args:
+        start (str): Tag, Branch or SHA1 of start point
+        stop (str):  Tag, Branch or SHA1 of end point, defaults to HEAD
+
+    Returns:
+        list[Log]: List of Logs
+
+    """
+    group_sep = '\x1D'
+    git_log_cmd = \
+        'git log -z --no-color --pretty=format:"%H{0}%aN{0}%aE{0}%B' \
+            .format(group_sep).split(' ')
+    git_log_cmd.append('%s..%s' % (start, stop))
+    out = subprocess.check_output(git_log_cmd).strip().split('\x00')
+    if len(out) == 1 and out[0] == '':
+        return []
+    else:
+        return [Log(*line.strip().split(group_sep)) for line in out]
+
+
+def _extract_issues(logs):
+    """Extract references to issues out of a list of Logs
+
+    Args:
+        logs (list[Log]): List of Logs to parse
+
+    Returns:
+        set[str]: Set of found issues as links to Github
+
+    """
+    issue_regex = re.compile(r'#([0-9]{4})')
+    issues = issue_regex.findall(" ".join([log.message for log in logs]))
+    base_url = 'https://github.com/oppia/oppia/issues/%s'
+    links = {base_url % issue for issue in issues}
+    return links
+
+
+def _check_versions(current_release):
+    """Checks if the versions for the exploration or collection schemas have
+    changed.
+
+    Args:
+        current_release (str): The current release tag to diff against.
+
+    Returns:
+        Dictionary consisting of name --> boolean that indicates whether or not
+            it has changed.
+    """
+    feconf_vars = ['CURRENT_EXPLORATION_STATES_SCHEMA_VERSION',
+                   'CURRENT_COLLECTION_SCHEMA_VERSION']
+    feconf_changed_version = {}
+    git_show_cmd = ('git show %s:feconf.py' % current_release).split()
+    old_feconf = subprocess.check_output(git_show_cmd)
+    with open('feconf.py', 'r') as feconf:
+        new_feconf = feconf.read()
+    for variable in feconf_vars:
+        version_re = r'%s\s*=\s*(\d+|\.)+'
+        old_version = re.findall(version_re % variable, old_feconf)[0]
+        new_version = re.findall(version_re % variable, new_feconf)[0]
+        feconf_changed_version[variable] = old_version != new_version
+    return feconf_changed_version
+
+
+def _git_diff_names_only(left, right='HEAD'):
+    """ Get names of changed files from git.
+
+    Args:
+        left (str): Lefthand timepoint
+        right (str): rightand timepoint
+
+    Returns:
+        (list): List of files that are different between the two points.
+
+    """
+    git_diff_cmd = ('git diff --name-only %s %s' % (left, right)).split()
+    return subprocess.check_output(git_diff_cmd).splitlines()
+
+
+def _check_setup_scripts(current_release, changed_only=False):
+    """Check if setup scripts have changed.
+
+    Args:
+        current_release (str): The current release tag to diff against.
+        changed_only (bool): If set to True returns only changed scripts
```

Comment:
I've changed the default to what we're actually using and modified the docstring to be more explicit